DEVELOP.md

```diff
@@ -30,6 +27,37 @@ python3 -m pip install --user --upgrade twine
 python3 -m twine upload dist/*
 ```
 
+### R
+
+From the command prompt, navigate to `src/r`. To install the package, open R and run:
+
+```r
+source("tools/deploy.R")
+```
+
+It will copy LICENSE, README.md from the main repo into `src/r`, create package documentation and install psychrolib for you.
+
+If you are a package developer, you can get more options by running:
+
+- On Windows:
+  ```
+  Rscript.exe tools/deploy.R [OPTION]
```

Comment:
If this is in your path are you sure you need to add the file extension to Rscript? If not please merge this and the Linux one into one for clarity.

Reply:
Nice catch. Will merge into one.